<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Y6FY0CN7TN-nagarajans" failures="0" tests="4" name="com.test.DisabilityCalculatorExecution" time="1.438" errors="1" timestamp="2024-06-12T16:46:22 IST" skipped="3">
  <testcase classname="com.test.DisabilityCalculatorExecution" name="OpenBrowser" time="1.438">
    <error message="Cannot invoke &quot;org.openqa.selenium.WebDriver.manage()&quot; because &quot;this.driver&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.manage()" because "this.driver" is null
at com.utilities.WaitUtils.implicitWaitCommand(WaitUtils.java:35)
at com.utilities.BaseUtils.OpenBrowser(BaseUtils.java:58)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:393)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:326)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:180)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
]]>
    </error>
  </testcase> <!-- OpenBrowser -->
  <testcase classname="com.test.DisabilityCalculatorExecution" name="navigateToDisabilityCalculator" time="0.000">
    <skipped/>
  </testcase> <!-- navigateToDisabilityCalculator -->
  <testcase classname="com.test.DisabilityCalculatorExecution" name="validateMonthlyMbe" time="0.000">
    <skipped/>
  </testcase> <!-- validateMonthlyMbe -->
  <testcase classname="com.test.DisabilityCalculatorExecution" name="validateSickBankTpf" time="0.000">
    <skipped/>
  </testcase> <!-- validateSickBankTpf -->
</testsuite> <!-- com.test.DisabilityCalculatorExecution -->
